charts/consul/test/unit/connect-inject-clusterrole.bats

```diff
@@ -29,6 +29,94 @@ load _helpers
       .
 }
 
+#--------------------------------------------------------------------
+# rules
+
+@test "connectInject/ClusterRole: sets get, list, and watch access to pods, endpoints, services, and namespaces in all api groups" {
```

Comment:
added these two tests that were missing from existing code, but are unrelated to webhook certs.